### PR TITLE
Add docformatter in pre-commit

### PR DESCRIPTION
**Pull request by edgarriba**
### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>

  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?

</details>

### CODE REVIEW

File: kornia/feature/affine_shape.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def __init__(self, pretrained: bool = False):
 
     @staticmethod
     def _normalize_input(x: torch.Tensor, eps: float = 1e-6) -> torch.Tensor:
-        "Utility function that normalizes the input by batch." ""
+        """Utility function that normalizes the input by batch.""" ""
```
Comment (edgarriba) [inline kornia/feature/affine_shape.py:189]:
```suggestion
        """Utility function that normalizes the input by batch."""
```

File: kornia/feature/hardnet.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def __init__(self, pretrained: bool = False) -> None:
 
     @staticmethod
     def _normalize_input(x: torch.Tensor, eps: float = 1e-6) -> torch.Tensor:
-        "Utility function that normalizes the input by batch." ""
+        """Utility function that normalizes the input by batch.""" ""
```
Comment (edgarriba) [inline kornia/feature/hardnet.py:72]:
```suggestion
        """Utility function that normalizes the input by batch."""
```

File: kornia/feature/hardnet.py [hunk 3]
```diff
@@ -155,7 +155,7 @@ def weights_init(m):
 
     @staticmethod
     def _normalize_input(x: torch.Tensor, eps: float = 1e-7) -> torch.Tensor:
-        "Utility function that normalizes the input by batch." ""
+        """Utility function that normalizes the input by batch.""" ""
```
Comment (edgarriba) [inline kornia/feature/hardnet.py:158]:
```suggestion
        """Utility function that normalizes the input by batch."""
```

**Review by Borda — APPROVED** [review]

**Comment by cclauss** [comment]
What is our flake8 `max-line-length`?  88 or something higher?

**Comment by edgarriba** [comment]
> What is our flake8 `max-line-length`? 88 or something higher?

120

**Review by ducha-aiki — APPROVED** [review]
LGTM

File: examples/depth_regression/main.py [hunk 4]
```diff
@@ -62,7 +62,8 @@ def load_image(file_name):
 
 
 def clip_and_convert_tensor(tensor):
-    """convert the input torch.Tensor to OpenCV image,clip it to be between
+    """convert the input torch.Tensor to OpenCV image,clip it to be between.
```
Comment (cclauss) [inline examples/depth_regression/main.py:65]:
```suggestion
    """Convert the input torch.Tensor to OpenCV image,clip it to be between.
```
;-)

**Review by cclauss — APPROVED** [review]
LGTM with some OPTIONAL comments.

File: examples/depth_regression/main.py [hunk 5]
```diff
@@ -29,7 +29,7 @@ def load_data(root_path, sequence_name, frame_id):
 
 
 def load_depth(file_name):
-    """Loads the depth using the sintel SDK and converts to torch.Tensor"""
+    """Loads the depth using the sintel SDK and converts to torch.Tensor."""
```
Comment (edgarriba) [inline examples/depth_regression/main.py:32]:
```suggestion
    """Load the depth using the sintel SDK and converts to torch.Tensor."""
```

File: examples/depth_regression/main.py [hunk 6]
```diff
@@ -49,7 +49,7 @@ def load_camera_data(file_name):
 
 
 def load_image(file_name):
-    """Loads the image with OpenCV and converts to torch.Tensor"""
+    """Loads the image with OpenCV and converts to torch.Tensor."""
```
Comment (edgarriba) [inline examples/depth_regression/main.py:52]:
```suggestion
    """Load the image with OpenCV and converts to torch.Tensor."""
```

File: examples/depth_warper/main.py [hunk 7]
```diff
@@ -9,7 +9,7 @@
 
 
 def load_depth(file_name):
-    """Loads the depth using the syntel SDK and converts to torch.Tensor"""
+    """Loads the depth using the syntel SDK and converts to torch.Tensor."""
```
Comment (edgarriba) [inline examples/depth_warper/main.py:12]:
```suggestion
    """Load the depth using the syntel SDK and converts to torch.Tensor."""
```

File: examples/depth_warper/main.py [hunk 8]
```diff
@@ -29,7 +29,7 @@ def load_camera_data(file_name):
 
 
 def load_image(file_name):
-    """Loads the image with OpenCV and converts to torch.Tensor"""
+    """Loads the image with OpenCV and converts to torch.Tensor."""
```
Comment (edgarriba) [inline examples/depth_warper/main.py:32]:
```suggestion
    """Load the image with OpenCV and converts to torch.Tensor."""
```

File: examples/homography_regression/main.py [hunk 9]
```diff
@@ -19,7 +19,7 @@ def load_homography(file_name):
 
 
 def load_image(file_name):
-    """Loads the image with OpenCV and converts to torch.Tensor"""
+    """Loads the image with OpenCV and converts to torch.Tensor."""
```
Comment (edgarriba) [inline examples/homography_regression/main.py:22]:
```suggestion
    """Load the image with OpenCV and converts to torch.Tensor."""
```

File: kornia/feature/laf.py [hunk 10]
```diff
@@ -524,8 +522,8 @@ def laf_is_inside_image(laf: torch.Tensor, images: torch.Tensor, border: int = 0
 
 
 def laf_to_three_points(laf: torch.Tensor):
-    """Converts local affine frame(LAF) to alternative representation: coordinates of
-    LAF center, LAF-x unit vector, LAF-y unit vector.
+    """Converts local affine frame(LAF) to alternative representation: coordinates of LAF center, LAF-x unit
```
Comment (edgarriba) [inline kornia/feature/laf.py:525]:
```suggestion
    """Convert local affine frame(LAF) to alternative representation: coordinates of LAF center, LAF-x unit
```

File: kornia/geometry/camera/pinhole.py [hunk 11]
```diff
@@ -487,15 +487,14 @@ def scale_pinhole(pinholes: torch.Tensor, scale: torch.Tensor) -> torch.Tensor:
 
 
 def get_optical_pose_base(pinholes: torch.Tensor) -> torch.Tensor:
-    """Get extrinsic transformation matrices for pinholes
+    """Get extrinsic transformation matrices for pinholes.
```
Comment (edgarriba) [inline kornia/geometry/camera/pinhole.py:490]:
```suggestion
    """Compute extrinsic transformation matrices for pinholes.
```

File: kornia/geometry/depth.py [hunk 12]
```diff
@@ -258,11 +258,10 @@ def _compute_projection(self, x, y, invd):
         return torch.cat([x, y], 1)
 
     def compute_subpixel_step(self) -> torch.Tensor:
-        """This computes the required inverse depth step to achieve sub pixel
-        accurate sampling of the depth cost volume, per camera.
+        """This computes the required inverse depth step to achieve sub pixel accurate sampling of the depth cost
```
Comment (edgarriba) [inline kornia/geometry/depth.py:261]:
```suggestion
        """Compute the required inverse depth step to achieve sub pixel accurate sampling of the depth cost
```

File: kornia/geometry/depth.py [hunk 13]
```diff
@@ -276,10 +275,9 @@ def compute_subpixel_step(self) -> torch.Tensor:
     def warp_grid(self, depth_src: torch.Tensor) -> torch.Tensor:
         """Computes a grid for warping a given the depth from the reference pinhole camera.
```
Comment (edgarriba) [inline kornia/geometry/depth.py:276]:
```suggestion
        """Compute a grid for warping a given the depth from the reference pinhole camera.
```

File: kornia/testing/__init__.py [hunk 14]
```diff
@@ -22,19 +20,20 @@ def xla_is_available() -> bool:
 
 # TODO: Isn't this function duplicated with eye_like?
 def create_eye_batch(batch_size, eye_size, device=None, dtype=None):
-    """Creates a batch of identity matrices of shape Bx3x3"""
+    """Creates a batch of identity matrices of shape Bx3x3."""
```
Comment (edgarriba) [inline kornia/testing/__init__.py:23]:
```suggestion
    """Create a batch of identity matrices of shape Bx3x3."""
```

File: kornia/testing/__init__.py [hunk 15]
```diff
@@ -22,19 +20,20 @@ def xla_is_available() -> bool:
...
     return torch.eye(eye_size, device=device, dtype=dtype).view(1, eye_size, eye_size).expand(batch_size, -1, -1)
 
 
 def create_random_homography(batch_size, eye_size, std_val=1e-3):
-    """Creates a batch of random homographies of shape Bx3x3"""
+    """Creates a batch of random homographies of shape Bx3x3."""
```
Comment (edgarriba) [inline kornia/testing/__init__.py:28]:
```suggestion
    """Create a batch of random homographies of shape Bx3x3."""
```

File: kornia/testing/__init__.py [hunk 16]
```diff
@@ -22,19 +20,20 @@ def xla_is_available() -> bool:
...
     std = torch.FloatTensor(batch_size, eye_size, eye_size)
     eye = create_eye_batch(batch_size, eye_size)
     return eye + std.uniform_(-std_val, std_val)
 
 
 def tensor_to_gradcheck_var(tensor, dtype=torch.float64, requires_grad=True):
     """Converts the input tensor to a valid variable to check the gradient.
```
Comment (edgarriba) [inline kornia/testing/__init__.py:35]:
```suggestion
    """Convert the input tensor to a valid variable to check the gradient.
```

File: kornia/testing/__init__.py [hunk 17]
```diff
@@ -54,14 +53,14 @@ def check_is_tensor(obj):
 
 
 def create_rectified_fundamental_matrix(batch_size):
-    """Creates a batch of rectified fundamental matrices of shape Bx3x3"""
+    """Creates a batch of rectified fundamental matrices of shape Bx3x3."""
```
Comment (edgarriba) [inline kornia/testing/__init__.py:56]:
```suggestion
    """Create a batch of rectified fundamental matrices of shape Bx3x3."""
```

File: kornia/testing/__init__.py [hunk 18]
```diff
@@ -54,14 +53,14 @@ def check_is_tensor(obj):
...
     F_rect = torch.tensor([[0.0, 0.0, 0.0], [0.0, 0.0, -1.0], [0.0, 1.0, 0.0]]).view(1, 3, 3)
     F_repeat = F_rect.repeat(batch_size, 1, 1)
     return F_repeat
 
 
 def create_random_fundamental_matrix(batch_size, std_val=1e-3):
-    """Creates a batch of random fundamental matrices of shape Bx3x3"""
+    """Creates a batch of random fundamental matrices of shape Bx3x3."""
```
Comment (edgarriba) [inline kornia/testing/__init__.py:63]:
```suggestion
    """Create a batch of random fundamental matrices of shape Bx3x3."""
```

File: kornia/testing/__init__.py [hunk 19]
```diff
@@ -95,7 +94,7 @@ def test_module(self):
 
 
 def cartesian_product_of_parameters(**possible_parameters):
-    """Creates cartesian product of given parameters"""
+    """Creates cartesian product of given parameters."""
```
Comment (edgarriba) [inline kornia/testing/__init__.py:97]:
```suggestion
    """Create cartesian product of given parameters."""
```

File: test/enhance/test_zca.py [hunk 20]
```diff
@@ -76,11 +72,7 @@ def test_identity(self, input_shape, eps, device, dtype):
         assert_close(data, data_hat, rtol=tol_val, atol=tol_val)
 
     def test_grad_zca_individual_transforms(self, device, dtype):
-        """
-
-        Checks if the gradients of the transforms are correct w.r.t to the input data
-
-        """
+        """Checks if the gradients of the transforms are correct w.r.t to the input data."""
```
Comment (edgarriba) [inline test/enhance/test_zca.py:75]:
```suggestion
        """Check if the gradients of the transforms are correct w.r.t to the input data."""
```
